### PR TITLE
Add the feature of adding employees

### PR DESCRIPTION
**Pull request by Grappler2010**
1. Add the employee-query in getQueries.ts.
2. Change styles for team-section.
3. Add sorting for employees.

### CODE REVIEW

File: app/components/TeamSection/index.tsx [hunk 1]
```diff
@@ -27,45 +29,107 @@ type TProps = {
...
+                      <P>{item.specialty}</P>
+                    </div>
+                  </div>
+                  <div className={styles.textContainer}>
+                    {
+                      item.firstSubtitle ?
+                        <P>{item.firstSubtitle}</P>
+                        :
+                        null
+                    }
```
Comment (dealwith) [inline app/components/TeamSection/index.tsx]:
```suggestion
                    {item.firstSubtitle && <P>{item.firstSubtitle}</P>
```

File: app/components/TeamSection/index.tsx [hunk 2]
```diff
@@ -27,45 +29,107 @@ type TProps = {
...
-            </div>
-          </div>
           {
             isMobile ?
-              <div className={styles.textContainer}>
-                <P>{data.items[0].secondSubtitle}</P>
-                <P>{data.items[0].thirdSubtitle}</P>
-                <P>{data.items[0].fourthSubtitle}</P>
-              </div>
+              sortedEmployees &&
```
Comment (dealwith) [inline app/components/TeamSection/index.tsx]: please read this principle and refactor this new code https://en.wikipedia.org/wiki/Don%27t_repeat_yourself

**Review by dealwith — CHANGES_REQUESTED** [review]

File: app/components/TeamSection/index.tsx (hunk 1 above)
Comment (dealwith) [reply app/components/TeamSection/index.tsx]: do the same for the code below until line 86

**Review by dealwith — CHANGES_REQUESTED** [review]

File: app/components/TeamSection/index.tsx (hunk 1 above)
Comment (Grappler2010) [reply app/components/TeamSection/index.tsx]: Done

File: app/components/TeamSection/index.tsx (hunk 2 above)
Comment (Grappler2010) [reply app/components/TeamSection/index.tsx]: Refactored this code

**Pull request merged**